### PR TITLE
feat: Set package and in app status for node frames

### PR DESCRIPTION
**Pull request by phacops**

### CODE REVIEW

**Review by Zylphrex — APPROVED** [review]

File: internal/occurrence/occurrence.go [hunk 1]
```diff
@@ -142,7 +142,7 @@ func NewOccurrence(p profile.Profile, ni nodeInfo) *Occurrence {
 		"frame_duration_ns":   ni.Node.DurationNS,
 		"frame_module":        ni.Node.Frame.Module,
 		"frame_name":          ni.Node.Name,
-		"frame_package":       ni.Node.Frame.Package,
+		"frame_package":       ni.Node.Frame.ModuleOrPackage(),
```
Comment (Zylphrex) [inline internal/occurrence/occurrence.go:145]: Why is this changing?

File: internal/sample/sample.go [hunk 2]
```diff
@@ -615,3 +616,17 @@ func (t *Trace) trimCocoaStacks() {
...
 	}
 }
+
+func (t *Trace) normalizeNodeFrames() {
+	packageInfo := make(map[string]packageutil.PackageInfo)
+	for i := 0; i < len(t.Frames); i++ {
+		f := t.Frames[i]
+		pi, exists := packageInfo[f.Path]
+		if !exists {
+			pi = packageutil.ParseNodeModuleFromPath(f.Path)
```
Comment (Zylphrex) [inline internal/sample/sample.go:626]: did you mean to add this to the `packageInfo` map but forgot?

File: internal/occurrence/occurrence.go (hunk 1 above)
Comment (phacops) [reply internal/occurrence/occurrence.go:145]: It shouldn't indeed.

File: internal/sample/sample.go (hunk 2 above)
Comment (phacops) [reply internal/sample/sample.go:626]: Yes.

**Pull request merged**